example/lib/pages_with_tiled_images/image_cache.dart

```diff
@@ -474,9 +474,14 @@ class DocumentPageTileImagePainter implements ImagePainter {
   Future<void> _paintImage(PageRegionRenderRequest request, TileIndex tileIndex) async {
     final imageCompleter = Completer<ui.Image>();
     final index = (tileIndex.col + tileIndex.row) % _imageAssets.length;
-    final imageStream = AssetImage(_imageAssets[index]).resolve(
+    final imageStream = ResizeImage(
+      AssetImage(_imageAssets[index]),
+      width: 1024,
+      height: 768,
```

Comment:
This is almost certainly still way too big, but should be better than the current implementation.

Ideally these sizes would be as close as possible to the resolution you will _actually_ display the image at.

Reply:
You also probably only want to specify one dimension or use `ResizeImagePolicy.fit` (which is not the default)

Reply:
Ah, I wasn't sure what values to use so I just chose that. When I tried it out, I got extra vertical whitespace.

Reply:
Yeah, if you use `fit` it will use that as a bounding box - if you specify one dimension it'll automatically scale the other dimension to preserve the aspect ratio.

Reply:
Gotcha, I switched it to the fit policy.